install.sh

```diff
@@ -6,36 +6,37 @@ RELEASE="$(rpm -E %fedora)"
 KERNEL_SUFFIX=""
 QUALIFIED_KERNEL="$(rpm -qa | grep -P 'kernel-(|'"$KERNEL_SUFFIX"'-)(\d+\.\d+\.\d+)' | sed -E 's/kernel-(|'"$KERNEL_SUFFIX"'-)//')"
 
-RPMFUSION_MIRROR_RPMS="https://mirrors.rpmfusion.org"
-if [ -n "${RPMFUSION_MIRROR}" ]; then
-    RPMFUSION_MIRROR_RPMS=${RPMFUSION_MIRROR}
-fi
+RPMFUSION_MIRROR_RPMS="${RPMFUSION_MIRROR:-https://mirrors.rpmfusion.org}"
 
-curl -Lo /tmp/rpms/rpmfusion-free-release-"${RELEASE}".noarch.rpm "${RPMFUSION_MIRROR_RPMS}"/free/fedora/rpmfusion-free-release-"${RELEASE}".noarch.rpm
-curl -Lo /tmp/rpms/rpmfusion-nonfree-release-"${RELEASE}".noarch.rpm "${RPMFUSION_MIRROR_RPMS}"/nonfree/fedora/rpmfusion-nonfree-release-"${RELEASE}".noarch.rpm
+declare -a REPO_PACKAGES=(
+    fedora-repos-archive
+    "${RPMFUSION_MIRROR_RPMS}/free/fedora/rpmfusion-free-release-${RELEASE}.noarch.rpm"
+    "${RPMFUSION_MIRROR_RPMS}/nonfree/fedora/rpmfusion-nonfree-release-${RELEASE}.noarch.rpm"
+)
```

Comment:
With the change to two `rpm-ostree install` commands a few lines below, this array variable is now unused and should be removed.

Reply:
Of course, i overlooked that, thanks.